lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -294,11 +295,19 @@ class FontSize extends Gatherer {
       .sort((a, b) => b.textLength - a.textLength)
       .slice(0, MAX_NODES_SOURCE_RULE_FETCHED)
       .map(async failingNode => {
-        failingNode.cssRule = await fetchSourceRule(driver, failingNode.node);
+        try {
+          const cssRule = await fetchSourceRule(driver, failingNode.node);
+          failingNode.cssRule = cssRule;
+        } catch (err) {
+          // The node was deleted. Don't set `cssRule` in that case.
+          log.error('font-size', err.message);
```

Comment:
This seems like this could lead to error fatigue, and is it anyone's error? `fetchSourceRule` already returns `undefined` if we can't find a font rule for the failing element, and we just silently drop that case. This seems basically equivalent in importance for logging


There's nothing to do in this case (and the element's not even around anymore). I could see maybe log.verbose, but there doesn't seem to be a good motivation for that (e.g.